common.yml

```diff
@@ -92,6 +92,7 @@ mongo_tools_variables:
       - name: integration-4.4-auth
       - name: integration-4.4-cluster
       - name: kerberos
+      - name: aws-auth
```

Comment:
when looking at [this task in the patchbuild|https://spruce.mongodb.com/task/mongo_tools_ubuntu1604_aws_auth_patch_db970fef5a038c6f47e8d765066f7a342858cecc_6047a313850e615d98d694bc_21_03_09_16_32_21/tests?execution=0], I noticed that there were a bunch of tests running in it that aren't actually AWS IAM tests.

It looks like this is because they don't call `SkipUnlessTestType` -- could we add the appropriate `SkipUnlessTestType` call to those tests so they don't clutter up the results for all the test tasks?

Reply:
Sure thing, they all look like unit tests to me.